.github/workflows/docker.yaml

```diff
@@ -28,164 +29,114 @@ jobs:
       packages: write
       contents: read
     steps:
-      - name: Checkout repository
+      -
```

Comment:
One last minor comment: could you start the first line at the dash (below too) to keep the formatting consistent with the other files?

Reply:
Done!